### PR TITLE
[Velo Wix Apps]Update wix app ID article

### PR DESCRIPTION
**Pull request by angeliquefm**
[Jira](https://jira.wixpress.com/browse/APIDOCS-383) task

### CODE REVIEW

File: guides/overview-apps.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-*   **[wix-pay-backend](https://www.wix.com/velo/reference/wix-pay-backend.html)**
-*   **[wix-events](https://www.wix.com/velo/reference/wix-events)**
-*   **[wix-events-backend](https://www.wix.com/velo/reference/wix-events-backend)**
-*   **[wix-bookings](https://www.wix.com/velo/reference/wix-bookings)**
-*   **[wix-bookings-backend](https://www.wix.com/velo/reference/wix-bookings-backend)**
-*   **[wix-paid-plans](https://www.wix.com/velo/reference/wix-paid-plans)**
-*   **[wix-paid-plans-backend](https://www.wix.com/velo/reference/wix-paid-plans-backend)**
-*   **[wix-pricing-plans-backend](https://www.wix.com/velo/reference/wix-pricing-plans-backend)**
\ No newline at end of file
+For Wix Apps by Wix, app data is returned in many of the API calls. Each solution is identified by its `appId`:
```
[the remark this answers is not in the document]
Comment (MartaBerman) [reply guides/overview-apps.md:5]: Anyway, regarding the sentence "For Wix Apps by Wix, app data is returned in many of the API calls." ... I am trying to put my finger on why, but I think we can do better. 

But let's "do better' in the other article. I don't think we need the text and the table here. I think we can just point to the other article. Let's keep the first paragraph delete the rest.

**Review by MartaBerman — CHANGES_REQUESTED** [review]

File: guides/overview-apps.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
 # Wix App APIs
 
-Customize Wix Apps with Velo APIs - including Stores, Bookings, Events, and Pay - to add even more features and functionality to your site. Combine APIs to create a unique app experience for your users.
```
[the remark this answers is not in the document]
Comment (MartaBerman) [reply guides/overview-apps.md:3]: @angeliquefm 

If you feel you handled this leftover comment from last time, please click "Resolve conversation."

File: guides/overview-apps.md [hunk 3]
```diff
@@ -1,15 +1,3 @@
-# Wix App APIs
+# Apps made by Wix
```
Comment (MartaBerman) [inline guides/overview-apps.md:1]: Our style is that for titles of articles, we capitalize the main words (not prepositions). 
For any other heading level, we use sentence case. 

So this line should be: "Apps Made by Wix" 

See the [style guide here](https://wix-private.github.io/tw-cafe/style-guide/grammar-formatting#headings).

File: guides/overview-apps.md [hunk 4]
```diff
@@ -1,15 +1,3 @@
...
-*   **[wix-pay-backend](https://www.wix.com/velo/reference/wix-pay-backend.html)**
-*   **[wix-events](https://www.wix.com/velo/reference/wix-events)**
-*   **[wix-events-backend](https://www.wix.com/velo/reference/wix-events-backend)**
-*   **[wix-bookings](https://www.wix.com/velo/reference/wix-bookings)**
-*   **[wix-bookings-backend](https://www.wix.com/velo/reference/wix-bookings-backend)**
-*   **[wix-paid-plans](https://www.wix.com/velo/reference/wix-paid-plans)**
-*   **[wix-paid-plans-backend](https://www.wix.com/velo/reference/wix-paid-plans-backend)**
-*   **[wix-pricing-plans-backend](https://www.wix.com/velo/reference/wix-pricing-plans-backend)**
\ No newline at end of file
+[Apps made by Wix](https://dev.wix.com/docs/develop-websites) offer a range of powerful business solutions that extend the capabilities of existing Wix products and introduce new functionalities. A site owner can install one or more of these solutions on their site and integrate them with Velo API's to ensure accurate data management and enhanced customization. Each app is identified by a unique `appId`.
```
Comment (MartaBerman) [inline guides/overview-apps.md:3]: I think you should put in your link with fake placeholder text already, so I can see how you intend to mention the link.

Comment (MartaBerman) [reply guides/overview-apps.md:3]: This sounds very negative: "to ensure accurate data management" ... it sounds like if you don't use these business solutions or you don't use Velo,  you cannot trust that your data is accurate.  

Also, no need to reinvent the wheel.  Just copy Rebecca's 1st paragraph "as is."  I see some issues with her first paragraph also, but at least we will be consistent.

Comment (MartaBerman) [reply guides/overview-apps.md:3]: Then I would add a 2nd paragraph that starts with your sentence  (but obviously we have to update the actual dummy link I put there for data quotas, as I know you know):  

Each app is identified by a unique `appId`. See a complete list of Made by Wix app IDs [here](https://dev.wix.com/docs/develop-websites/articles/databases/about-data-quotas).

**Review by MartaBerman — CHANGES_REQUESTED** [review]

File: guides/overview-apps.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (angeliquefm) [reply guides/overview-apps.md]: Yes, it is from Rebecca's article.

File: guides/overview-apps.md (hunk 4 above)
Comment (angeliquefm) [reply guides/overview-apps.md]: I'm very confused about what you want here. 
1. placeholder text: I have added placeholder text [Apps made by Wix].
2. The last time we met, you said that you didn't like Rebecca's article, and that it was only a half truth that apps are used to manage data.

**Review by MartaBerman — APPROVED** [review]

**Pull request merged**